com.markopejic.downloader.json

```diff
@@ -0,0 +1,120 @@
+{
+    "id": "com.markopejic.downloader",
```

Comment:
`markopejic.com` lead to a 403 forbidden.

Reply:
Fixed the 403 forbidden error, the website is now reachable.